tests/storage/data_import_cron/conftest.py

```diff
@@ -0,0 +1,61 @@
+import pytest
+from ocp_resources.data_import_cron import DataImportCron
+
+from utilities.constants import BIND_IMMEDIATE_ANNOTATION, WILDCARD_CRON_EXPRESSION, Images
+from utilities.infra import create_ns
+from utilities.storage import create_dv
+
+LAST_IMPORT_IS_UP_TO_DATE_MESSAGE = "Latest import is up to date"
+
+
+@pytest.fixture(scope="class")
+def data_import_cron_pvc_target_namespace(unprivileged_client):
+    yield from create_ns(unprivileged_client=unprivileged_client, name="test-data-import-cron-pvc-target")
+
+
+@pytest.fixture(scope="class")
+def dv_vm_for_data_import_cron(namespace, storage_class_name_scope_module, rhel9_http_image_url):
+    with create_dv(
+        dv_name="dv-pvc-source-rhel",
+        namespace=namespace.name,
+        url=rhel9_http_image_url,
+        size=Images.Rhel.DEFAULT_DV_SIZE,
+        storage_class=storage_class_name_scope_module,
+    ) as dv:
+        yield dv
+
+
+@pytest.fixture(scope="class")
+def data_import_cron_pvc_source(
+    unprivileged_client,
+    data_import_cron_pvc_target_namespace,
+    storage_class_name_scope_module,
+    dv_vm_for_data_import_cron,
+    storage_class_matrix__module__,
+):
+    with DataImportCron(
+        name="pvc-import-cron",
+        namespace=data_import_cron_pvc_target_namespace.name,
+        schedule=WILDCARD_CRON_EXPRESSION,
+        managed_data_source="pvc-import-datasource-pvc-source",
+        annotations=BIND_IMMEDIATE_ANNOTATION,
+        template={
+            "spec": {
+                "source": {
+                    "pvc": {
+                        "name": dv_vm_for_data_import_cron.name,
+                        "namespace": dv_vm_for_data_import_cron.namespace,
+                    }
+                },
+                "sourceFormat": "pvc",
+                "volumeMode": storage_class_matrix__module__[storage_class_name_scope_module]["volume_mode"],
+                "storage": {
+                    "storageClassName": storage_class_name_scope_module,
+                    "accessModes": [storage_class_matrix__module__[storage_class_name_scope_module]["access_mode"]],
+                    "resources": {"requests": {"storage": dv_vm_for_data_import_cron.size}},
```

Comment:
Please look at the DataImportCron resource in your cluster and remove the fields that are not needed or do not exist in the DataImportCron spec 

(you can check by running `oc explain DataImportCron.spec.template.spec.<field>`)